db/seeds.rb

```diff
@@ -5,18 +5,25 @@
+second_user = User.create(name: 'John Smith', photo: 'https://images.unsplash.com/photo-1508921912186-1d1a45ebb3c1', bio: 'Doctor from Brazil.')
+third_user = User.create(name: 'Noah', photo: 'https://images.unsplash.com/photo-1508921912186-1d1a45ebb3c1', bio: 'Pilot from Germany.')
+fourth_user = User.create(name: 'Ann Charlotte', photo: 'https://images.unsplash.com/photo-1508921912186-1d1a45ebb3c1', bio: 'Lawyer from Ghana.')
 
 # Posts
-first_post = Post.create(author_id: first_user.id, title: 'Hello', text: 'This is my first post')
-second_post = Post.create(author_id: first_user.id, title: 'Hi', text: 'This is my second post')
-third_post = Post.create(author_id: first_user.id, title: 'Hi there', text: 'This is my third post')
-fourth_post = Post.create(author_id: first_user.id, title: 'Hello!', text: 'This is my fourth post')
+first_post = Post.create(author: first_user, title: 'Hello', text: 'This is my first post')
+second_post = Post.create(author: first_user, title: 'Hi', text: 'This is my second post')
+third_post = Post.create(author: first_user, title: 'Hi there', text: 'This is my third post')
+fourth_post = Post.create(author: first_user, title: 'Hello!', text: 'This is my fourth post')
 
 # Comments
-Comment.create(post_id: first_post.id, author_id: first_user.id, text: 'Hi Tom!')
-Comment.create(post_id: first_post.id, author_id: first_user.id, text: 'Hi Tom!')
-Comment.create(post_id: first_post.id, author_id: first_user.id, text: 'Hi!')
-Comment.create(post_id: first_post.id, author_id: first_user.id, text: 'Hello there')
-Comment.create(post_id: first_post.id, author_id: first_user.id, text: 'Hello there!')
-Comment.create(post_id: first_post.id, author_id: first_user.id, text: 'Hello')
+Comment.create(author_id: first_user, text: 'Lorem officia deserunt mollit anim id est laborum.', posts_id: first_post)
+Comment.create(posts_id: first_post, author_id: second_user, text: 'Hi Tom!')
+Comment.create(posts_id: first_post, author_id: first_user, text: 'Great Post')
+Comment.create(posts_id: second_post, author_id: fourth_user, text: 'Nice Content, Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.')
+Comment.create(posts_id: third_post, author_id: third_user, text: 'Hello there!')
+Comment.create(posts_id: first_post, author_id: first_user, text: 'Hi, Lorem ipsum dolor sit amet')
+Comment.create(posts_id: first_post, author_id: first_user, text: 'Hello, Lorem ipsum dolor sit amet')
+Comment.create(posts_id: fourth_post, author_id: first_user, text: 'Like the post, Lorem ipsum dolor sit amet')
```

Comment:
- [Optional] Let's use `post_id` which is in our database instead of `posts_id` otherwise `rails db:seed` will fail

Reply:
Oh, true! Nice observation. Those small things can make or break a program. Thank you so much.

Reply:
But it's strange, because I tested here and posts_id works in my db:seed, and post_id doesn't...

Reply:
![image](https://user-images.githubusercontent.com/90217257/172748225-abb1e1eb-2d40-4fbb-9198-a1da9c964c9f.png)